packages/account/src/BiconomySmartAccountV2.ts

```diff
@@ -257,13 +258,105 @@ export class BiconomySmartAccountV2 extends BaseSmartContractAccount {
     return this.accountAddress;
   }
 
+  /**
+   * Transfers funds from Smart Account to recipient (usually EOA)
+   * @param recipient - Address of the recipient
+   * @param withdrawalRequests - Array of withdrawal requests {@link WithdrawalRequest}. If withdrawal request is an empty array, it will transfer the balance of the native token. Using a paymaster will ensure no dust remains in the smart account.
+   * @param buildUseropDto - Optional. {@link BuildUserOpOptions}
+   *
+   * @returns Promise<UserOpResponse> - An object containing the status of the transaction.
+   *
+   * @example
+   * import { createClient } from "viem"
+   * import { createSmartAccountClient, NATIVE_TOKEN_ALIAS } from "@biconomy/account"
+   * import { createWalletClient, http } from "viem";
+   * import { polygonMumbai } from "viem/chains";
+   *
+   * const USDT = "0xda5289fcaaf71d52a80a254da614a192b693e977";
+   * const signer = createWalletClient({
+   *   account,
+   *   chain: polygonMumbai,
+   *   transport: http(),
+   * });
+   *
+   * const smartAccount = await createSmartAccountClient({ signer, bundlerUrl, biconomyPaymasterApiKey });
+   *
+   * const { wait } = await smartAccount.withdraw(
+   *  account.pubKey, // recipient
+   *  [
+   *    { address: USDT, amount: BigInt(1) },
+   *    { address: NATIVE_TOKEN_ALIAS, amount: BigInt(1) }
+   *  ],
+   *  {
+   *    paymasterServiceData: { mode: PaymasterMode.SPONSORED },
+   *  }
+   * );
+   *
+   * // OR to withdraw all of the native token, leaving no dust in the smart account
+   *
+   * const { wait } = await smartAccount.withdraw(account.pubKey, [], {
+   *  paymasterServiceData: { mode: PaymasterMode.SPONSORED },
+   * });
+   *
+   * const { success } = await wait();
+   */
+  public async withdraw(
+    recipient: Hex,
```

Comment:
we could also make recipient per each withdraw request

Reply:
optionally. if not provided use the main recipient given above

Reply:
They can always call withdraw multiple times too per recipient. Actually having a method named 'withdraw' that doesn't go to the owner is slightly confusing, but I couldn't think of a clear path to finding the recipient without it being provided by the dev. I would have hardcoded this if I could have... 

Can you think of a way to reliably find the owner in this case? I think it would be best now that I'm thinking about it...

Reply:
hmm thinking. we could have method single withdraw (which also acts as withdrawERC20) 
and also withdrawMultipleERC20 and give them an option to provide receiver for each withdrawal. 

these methods are risky though cause if we make mistake in sdk or choose to go rough then all tokens would be transferred to us

Reply:
I mean
```
[
   *    { address: USDT, amount: BigInt(1) }
```

here they could also provide recipient address, instead of having common recipient..or maybe if not provided then fallback to common recipient given above. 

I agree with you it could by default go to the owner in most cases where owner is another hot wallet metamask EOA. but they might want to send elsewhere or to a friend.

Reply:
OK makes sense. Will add.